test/auth.tests.js

```diff
@@ -113,7 +113,7 @@ describe('auth', function () {
         assert.equal(parsed.query.client_id, '__test_client_id__');
         assert.equal(parsed.query.scope, 'openid profile email');
         assert.equal(parsed.query.response_type, 'code');
-        assert.equal(parsed.query.response_mode, undefined);
+        assert.equal(parsed.query.response_mode, 'query');
```

Comment:
as per the request to not default the mode to an explicit value this needs to be reverted too.